integration_test/workflow_tests/test_packages/ingest/test_multiple_granules_happy_path.py

```diff
@@ -0,0 +1,220 @@
+                            "key": key_name_2,
+                        }
+                        ]
+                    }
+                    ],
+                    "copied_to_orca": [
+                    "s3://" + bucket_name + "/" + key_name_1,
+                    "s3://" + bucket_name + "/" + key_name_2
+                    ]
+                },
+                "meta": {
+                    "provider": {
+                    "id": provider_id,
+                    "name": provider_name
+                    },
+                    "collection": {
+                        "meta": {
+                            "orca": {
+                                "excludedFileExtensions":
+                                    excluded_filetype
+                            }
+                        },
+                        "name": collection_name,
+                        "version": collection_version
+                        }
+                },
+                "cumulus_meta": {
+                    "execution_name": execution_id
+                },
+                "task_config": {
+                    "excludedFileExtensions": "{$.meta.collection.meta.orca.excludedFileExtensions}",
+                    "s3MultipartChunksizeMb": "{$.meta.collection.meta.s3MultipartChunksizeMb}",
+                    "providerId": "{$.meta.provider.id}",
+                    "providerName": "{$.meta.provider.name}",
+                    "executionId": "{$.cumulus_meta.execution_name}",
+                    "collectionShortname": "{$.meta.collection.name}",
+                    "collectionVersion": "{$.meta.collection.version}",
+                    "defaultBucketOverride": "{$.meta.collection.meta.orca.defaultBucketOverride}",
+                    "defaultStorageClassOverride": "{$.meta.collection.meta.orca.defaultStorageClassOverride}"
+                },
+                "exception": "None"
+                }
+
+            execution_info = boto3.client("stepfunctions").start_execution(
+                stateMachineArn=helpers.orca_copy_to_archive_step_function_arn,
+                input=json.dumps(copy_to_archive_input, indent=4),
+            )
+
+            step_function_results = helpers.get_state_machine_execution_results(
+                execution_info["executionArn"]
+            )
+
+            self.assertEqual(
+                "SUCCEEDED",
+                step_function_results["status"],
+                "Error occurred while starting step function.",
+            )
+            self.assertEqual(
+                expected_output,
+                json.loads(step_function_results["output"]),
+                "Expected step function output not returned.",
+            )
+            # verify that the objects exist in recovery bucket
+            try:
+                for key in [key_name_1, key_name_2]:
+                    head_object_output = boto3.client("s3").head_object(Bucket=recovery_bucket_name, Key=key)
```

Comment:
This type of check should also be present for excluded files, to verify that they were NOT copied.

Reply:
added check to verify files do not exist